.github/workflows/Windows.yml

```
@@ -30,11 +30,11 @@
       run: pip install jinja2
     
     - name: Install Vulkan SDK
-      uses: humbletim/install-vulkan-sdk@v1.2
+      uses: humbletim/install-vulkan-sdk@033219e2d69dd4d3e873e217eb385fdd4b4438f4
       with:
-        version: latest
+        version: 1.4.321.0
         cache: true
-
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)